spec/integration/pgpapi.spec.js

```diff
@@ -3,15 +3,21 @@
 
 describe('PGP api integration', function() {
   'use strict';
-  var freedom;
+  var fdom, path;  // hack to avoid masking freedom when already defined
+  if (typeof freedom === 'undefined') {
+    fdom = require('freedom-for-node').freedom;
```

Comment:
I've tried this but unfortunately it seems like the only way for node to be sure that something exists is for it to have an explicit "var foo" declaration somewhere, but any "var freedom" declaration (even just in a conditional) ends up clobbering the chromeapp freedom. I'll go ahead and merge it as it is, but if you've got a tweak that works let me know - thanks!